### PR TITLE
Issue 1156 -- biology iconned features 

### PR DESCRIPTION
**Pull request by richelleyy**

### CODE REVIEW

File: styles/books/anatomy/book.scss [hunk 1]
```diff
@@ -1,15 +0,0 @@
-@import 'framework/framework';
```
Comment (brittany-johnson) [inline styles/books/anatomy/book.scss]: This file should not be deleted

File: styles/README.md [hunk 2]
```diff
@@ -1,323 +0,0 @@
-# Using the BakedStyles framework
```
Comment (brittany-johnson) [inline styles/README.md]: This file should not be deleted

File: styles/books/biology/book.scss [hunk 3]
```diff
@@ -6,6 +6,10 @@
```
Comment (brittany-johnson) [inline styles/books/biology/book.scss]: The chemistry folder should not be renamed to bio, a new bio folder should be created

File: styles/designs/carnival/parts/_bioiconfeature-components.scss [hunk 4]
```diff
@@ -0,0 +1,52 @@
+$FeatureContainer:(
```
Comment (helenemccarron) [inline styles/designs/carnival/parts/_bioiconfeature-components.scss]: Can you rename the feature file to omit bio? Those features potentially could be used in other books.

File: styles/designs/carnival/parts/_bioiconfeature-shapes.scss [hunk 5]
```diff
@@ -0,0 +1,49 @@
+@import 'bioiconfeature-components';
```
Comment (helenemccarron) [inline styles/designs/carnival/parts/_bioiconfeature-shapes.scss]: can we remove the book's name from the variable naming? As with the files, those could potentially be used in other books.

**Comment by tomjw64** [comment]
Good work @richelleyy

File: styles/books/biology/book.scss [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+@include use('EvolutionIcon','carnival:::Evolution');
+@include use('ScientificMethodIcon','carnival:::Scientific');
+@include use('VisualConnection','carnival:::Visual');
+@include use('Preface','carnival:::Preface');
+@include use('KeyTerm','carnival:::EocKeyTerm');
+@include use('KeyEquation','carnival:::EocKeyEqu');
+@include use('SectionSummary','carnival:::EocSummary');
+@include use('Exercise','carnival:::EocExercise');
+@include use('EverydayLife', 'carnival:::BorderFeatureShape');
+@include use('Portrait', 'carnival:::BorderFeatureShape');
```
[the remark this answers is not in the document]
Comment (richelleyy) [reply styles/books/biology/book.scss]: @brittany-johnson I copied and pasted from the `book.scss` from chemistry, but I'm not sure what settings can be applied to biology.

**Pull request merged**